CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 1.11
 
+## 1.11.2 - 2023-05-15
+
+### Changed
+
+- Update Open vSwitch to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Fix IPsec e2e flaky. ([#4774](https://github.com/antrea-io/antrea/pull/4774), [@xliuxu])
```

Comment:
No need to include non user-facing change

Reply:
Not addressed.